modules/rosa-understanding-deployment-modes.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * rosa_getting_started_sts/rosa_creating_a_cluster_with_sts/rosa-sts-creating-a-cluster-with-customizations.adoc
+
+:_content-type: CONCEPT
+[id="rosa-understanding-deployment-modes_{context}"]
+= Understanding the auto and manual deployment modes
+
+When installing a {product-title} (ROSA) cluster that uses the AWS Security Token Service (STS), you can choose between the `auto` and `manual` ROSA CLI (`rosa`) modes to create the required AWS Identity and Access Management (IAM) resources.
+
+`auto` mode:: With this mode, `rosa` immediately creates the required IAM roles and policies, and an OpenID Connect (OIDC) provider in your AWS account.
+
+`manual` mode:: With this mode, `rosa` outputs the `aws` commands needed to create the IAM resources. The corresponding policy JSON files are also saved to the current directory. By using `manual` mode, you can review the generated `aws` commands before running them manually. `manual` mode also enables you to pass the commands to another administrator or group in your organization so that they can create the resources.
+
```

Comment:
Suggestion...   Can we note an important tip here?..  when selecting manual mode, a cluster installation waits for the existence of the operator roles and oidc provider, and without creating them, a manual mode cluster will remain in waiting mode and not proceed through installation to completion.

Reply:
This is a great suggestion, thanks. I have subsequently added an important admonition.